.gitignore

```diff
@@ -1,4 +1,5 @@
 yarn.lock
+package-lock.json
```

Comment:
this change is meaningless, so probably makes sense to restore it to how it previously was to avoid noise in history

Reply:
It is not meaningless, please check all changes (I just moved it to the other lockfile ignore rule).

Reply:
This is better as the lockfile ignore rules are both at the top.

Reply:
Which is more consistent. To make it clear we can also add some comments.

Reply:
Yeah, I meant the ordering. And yes, it is more consistent, you are right.